### PR TITLE
Added the theme layout to the chatroom

### LINKED ISSUE #203: Add theme layout to chat page.
### Description
Add theme layout in order to have concurrency when going to the chat page. Change from the old theme to the new theme that we have created. 

### Expected Behavior
The theme will show as intended and is not the old layout.

### PR DESCRIPTION
**Pull request by jusasin**
### Issue Numbers
Closes #203 and address #90 

### Description 
The theme layout is now applied to the chatroom page so it looks like the rest of the pages. Got help from Blake and Macgyver to align the page and to protect the page as well. Also did some other maintenance work for this PR with issue #90 in regards to updating the URIs. 

### Testing
Test cases will be added after the whole page is done. I will take care of that. 

### Possible Errors
No possible errors 

### Pictures 
image

### CODE REVIEW

**Comment by jusasin** [comment]
The deployment fails because of some other test. We have talked about it and we will fix later just look at the changes that was made in the files.

**Comment by aiharc1** [comment]
Hi Justin, after reviewing this PR it seems that the entire deployment fails - not just the test cases. I apologize but I will have to hold off on accepting this PR until the deployment works at the least.

**Review by aiharc1 — COMMENTED** [review]
Due to the deployment failure I cannot accept at this time. It seems like it is beyond just the test cases

**Comment by jusasin** [comment]
This is really weird that my deployment fails because taking all the code, I am allowed to run and not run into any issues... I am really unsure as to why it is failing.

**Comment by jusasin** [comment]
Fixed my deployment issue, I pulled from the recently merged mains and it still seems that the tests are failing. We can continue to fix those tests soon but focusing on chat room components

**Comment by jusasin** [comment]
Fixed the test issues as well! WOOOHOO

**Review by tbg510 — APPROVED** [review]
Glad you were able to fix the seemingly random tests that were failing. We finally have the new layout for the chatroom, great work!

File: small-talk/tailwind.config.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 module.exports = {
   content: [
-    "./pages/**/*.{js,ts,jsx,tsx,mdx,json}",
-    "./components/**/*.{js,ts,jsx,tsx,mdx,json}",
-    "./app/**/*.{js,ts,jsx,tsx,mdx,json}",
-    "./src/**/*.{js,ts,jsx,tsx,mdx,json}",
+    "./pages/**/*.{js,ts,jsx,tsx,mdx}",
+    "./components/**/*.{js,ts,jsx,tsx,mdx}",
+    "./app/**/*.{js,ts,jsx,tsx,mdx}",
+    "./src/**/*.{js,ts,jsx,tsx,mdx}",
```
Comment (Redblaze74) [inline small-talk/tailwind.config.js:7]: Could you add the json files back to within the content, in the old layout it had issues reading CSS within classNames from JSON files due to this.

**Review by salazw1 — APPROVED** [review]
Good job on reformatting the chat room and fixing the initial issues of the test cases. It's great that our webpages are becoming more consistent in appearance!

File: small-talk/tailwind.config.js (hunk 1 above)
Comment (jusasin) [reply small-talk/tailwind.config.js]: Went ahead and fixed this, thank you for pointing this out!

**Review by Redblaze74 — APPROVED** [review]
LGTM, good job converting our main functionality into the new layout and formatting it so it looks nice.

**Comment by aiharc1** [comment]
Nice work closing the issues out! Keep up the good work

**Pull request merged**